### ISSUE TITLE
If ListView is created using the default constructor it cannot be focussed

### ISSUE THREAD

**Issue by timothyparez** [migueldeicaza/gui.cs#226]
If you create a `ListView` using the default constructor and set the source later:

```
var listView = new ListView();
listView.SetSource(new string[] { "batman", "robin", "alfred" });
```

then you cannot cycle through the items in the `ListView` even though they are visible.

If you create the `ListView` and set the source like this:

```
var listView = new ListView(new string[] {});
listView.SetSource(new string[] { "batman", "robin", "alfred" });
```

it works just fine.

**Comment by BarryNolte** [comment]
Looking at the source, the CanFocus property doesn't get set in some of the constructors.  I'm not sure that's by design or not, (e.g. Not loaded with data, doesn't need to take focus), because it appears inconsistently applied by default.  Another solution is to set the CanFocus property when needed.

**Comment by tig** [comment]
Does this still repro?

**Comment by BarryNolte** [comment]
I looked at the code again, and I would say it still does.  Looking at other controls, they seem to pass through other constructors if the default is called and setup the control (CanFocus being one flag in the group of defaults), but for ListView it doesn't.  I didn't spend enough time to see if there is a consistency issue between controls, or it's isolated to ListView.  This might be related to #517, there some possible code paths that look iffy.

**Comment by tig** [comment]
I'm punting this from `0.90`. We can fix this before `1.0`.
